OpenIPSL/Electrical/Renewables/PSSE/ElectricalController/BaseClasses/PIwithNoVariableLimiter.mo

```diff
@@ -0,0 +1,82 @@
+within OpenIPSL.Electrical.Renewables.PSSE.ElectricalController.BaseClasses;
+model PIwithNoVariableLimiter "PI with no variable limiter controller for WECC electrical controllers"
+  import Modelica.Units.SI;
+  parameter SI.PerUnit K_P "Voltage regulator proportional gain";
```

Comment:
Be careful with the units. I don't think that the controller gain here (and in many other places) is a per-unit value but rather a simple Real value.

Reply:
Yes you are right. I've fixed this issue.

Reply:
Make sure to fix them in all places of this package. :-)

Reply:
Completed!